databricks/koalas/tests/test_indexing.py

```diff
@@ -1196,11 +1196,18 @@ def test_series_iloc_setitem(self):
         pser = pdf.a
         kser = kdf.a
 
-        pser.iloc[[1]] = -pdf.b
-        kser.iloc[[1]] = -kdf.b
+        pser.iloc[[0, 1, 2]] = -pdf.b
+        kser.iloc[[0, 1, 2]] = -kdf.b
```

Comment:
what are these diff?

Reply:
The length of parameter to `iloc` should be the same with the length of the assigning values from pandas 1.1.2.

In short, `len([0, 1, 2])` and `len(-pdf.b)` should be the same.